docs/release-notes.rst

```diff
@@ -6,6 +6,95 @@
  Release Notes
 ###############
 
+**************
+ Version 0.20
+**************
+
+Version 0.20.1
+==============
+
+**Release Date:** March 15, 2023
+
+**Breaking Changes**
+
+-  Database: Several unused columns have been dropped from ``raw_steps``, ``raw_validations``,
+   ``raw_checkpoints`` database tables. The database migration will involve a sequential scan for
+   these tables, and it may take significant amount of time, depending on the database size and
+   performance.
+
+**New Features**
+
+-  Added an ability for tasks or experiments to expose arbitrary ports that can be tunneled to the
```

Comment:
```suggestion
-  Allow tasks and experiments to expose arbitrary ports that can be tunneled to via the
```
Not sure if something like "via" was the intent, or if there should instead be a noun at the end of the line. @ioga